students/EricAdams/session04/mailroom.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+
+"""
+Used Chris's solution to session 3 version and added to it to create this,
```

Comment:
fair enough -- but make sure you understand the code, and why i wrote it that way!

-CHB